lookout/style/typos/benchmarks/typo_commits_report.py

```diff
@@ -42,23 +44,33 @@ def run(self, ptr: ReferencePointer, data_service: DataService) -> Iterable[Typo
         """
         for file in request_files(data_service.get_data(), ptr, contents=True, uast=True,
                                   unicode=False):
-            if file.path == self.config["filepath_to_analyze"]:
+            if file.path == self.config["analyze"]["filepath"]:
                 break
         else:
-            raise ValueError("No such file %s in %s" % (self.config["filepath_to_analyze"], ptr))
-
-        typos_fixes = list(self.generate_typos_fixes([
-            UnicodeChange(head=file, base=File(path=file.path, language=file.language))]))
+            raise ValueError("No such file %s in %s" % (self.config["analyze"]["filepath"], ptr))
+        line = self.config["analyze"]["line"] + 1
+        with patch("lookout.style.typos.analyzer.find_new_lines") as mocked_find_new_lines, \
+                patch("lookout.style.typos.analyzer.uast2sequence") as mocked_uast2sequence:
+            mocked_find_new_lines.return_value = [line]
```

Comment:
This patches forces to check only one line.
Also, `mocked_uast2sequence` disables check if the identifier was already present in the dataset because we have only one commit for the analysis.

Reply:
This is a bit too hacky. We should make it a `@classmethod` inside the class then so that we can properly override it here.